### PR TITLE
Initial commit for freesurfer package

### PR DESCRIPTION
**Pull request by robgics**
Add package.py

### CODE REVIEW

File: var/spack/repos/builtin/packages/freesurfer/package.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    homepage = "http://freesurfer.net/"
+
+    # A license is required, but is free to obtain.
+    license_required = True
+    license_files = ["./license.txt"]
+
+    maintainers = ["robgics"]
+
+    def url_for_version(self, version):
+        return "http://surfer.nmr.mgh.harvard.edu/pub/dist/freesurfer/{0}/freesurfer-linux-centos7_x86_64-{1}.tar.gz".format(
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/freesurfer/package.py]: We prefer to use the `https` protocol.

```suggestion
        return "https://surfer.nmr.mgh.harvard.edu/pub/dist/freesurfer/{0}/freesurfer-linux-centos7_x86_64-{1}.tar.gz".format(
```

File: var/spack/repos/builtin/packages/freesurfer/package.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    homepage = "http://freesurfer.net/"
+
+    # A license is required, but is free to obtain.
+    license_required = True
+    license_files = ["./license.txt"]
+
+    maintainers = ["robgics"]
+
+    def url_for_version(self, version):
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/freesurfer/package.py]: Please move this method so it occurs after the last `depends_on` directive.

**Review by tldahlgren — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/freesurfer/package.py (hunk 2 above)
Comment (robgics) [reply var/spack/repos/builtin/packages/freesurfer/package.py]: Changes made and committed

**Review by tldahlgren — APPROVED** [review]
LGTM.  (Did not download the software to confirm the three sha256 though.)

**Pull request merged**